### ISSUE TITLE
After selecting some elements in web content while using a screen reader, it's not possible to land on them when exploring via touch [Bug]

### ISSUE THREAD

**Issue by DiegoARL38** [mozilla-mobile/fenix#3197]
## Steps to reproduce
1. Open Fenix with a screen reader on.
1. Go to any page with a text input.
1. Focus the text input and activate it.
1. Close the keyboard.
1. Explore via touch the screen.
    1. Expected behavior
It's possible to land on the text input.
    1. Actual behavior
It's not possible to land on the text input, although it is when using navigation gestures
    1. Device information

* Android device: TP-Link Neffos C5 Max (Android 5.1)
* Fenix version: Build #11561807

### Dependencies
* [ ] GV dismiss keyboard bug https://bugzilla.mozilla.org/show_bug.cgi?id=1587557#c6

### Notes
* This also applies to links; easy to verify in a page in which links are used to perform actions, like the column headers in Bugzilla.
* This applies to buttons, as well.
* Tested with both ShinePlus and TalkBack.

**Comment by DiegoARL38** [comment]
This behaviour is also present on normal Firefox for Android

**Comment by cpeterso** [comment]
Since this bug is also reproducible in Firefox for Android, I filed a new GV bug in Bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1558289

**Comment by DiegoARL38** [comment]
Updated to reflect the fact that this bug applies to buttons, too.

**Comment by asadotzler** [comment]
The fix for this landed in GV on 7/2 https://bugzilla.mozilla.org/show_bug.cgi?id=1558289#c8 but it seems we haven't updated GV in Fenix since 6/12.

**Comment by ekager** [comment]
From the Bugzilla issue it looks like a patch landed in 69 which Fenix now uses that will improve this, but there may be issues that remain.

**Comment by cpeterso** [comment]
I am removing the `needs:gv` and `waiting` labels because the GV fix landed in GV 69 Beta and Fenix is using GV 69 Beta. Someone just needs to test a recent Fenix build.

**Comment by sv-ohorvath** [comment]
I can still reproduce the problem, I can reach the text box but double tap doesn't activate it all the time. I tested on bugzilla.mozilla.org and on google search with a Pixel 3 (Android 9) and a Mi4i (Android 5)

**Comment by cpeterso** [comment]
> I can still reproduce the problem, I can reach the text box but double tap doesn't activate it all the time. I tested on bugzilla.mozilla.org and on google search with a Pixel 3 (Android 9) and a Mi4i (Android 5)

@eeejay, QA can still reproduce this screen reader bug in Fenix + GeckoView 70, even though the a11y [bug 1558289](https://bugzilla.mozilla.org/show_bug.cgi?id=1558289) was fixed in GeckoView 69. Do you think this is a different problem?

**Comment by sblatz** [comment]
I'm re-labeling this to p2. @eeejay please re-assign to p1 if you think this is critical!

**Comment by bifleming** [comment]
@cpeterso sounds like still a problem in GV 70

**Comment by cpeterso** [comment]
I filed a new GV bug for the problem in GV 70: https://bugzilla.mozilla.org/show_bug.cgi?id=1587557

**Comment by cpeterso** [comment]
@sv-ohorvath - Can you please re-test a recent Fenix 2.1 (GV 70) or Nightly (GV 71)?

Marco Zehe on the Gecko a11y team was able to reproduce the problem in Fennec 68.2 (as expected since the Gecko bug was supposedly fixed in Gecko 69), but he can't reproduce in Fenix 2.1 (built on 2019-09-23) or Fenix Nightly (GV 71.0a1 2019-10-14).

https://bugzilla.mozilla.org/show_bug.cgi?id=1558289#c30

**Comment by sv-ohorvath** [comment]
I made a video with the latest Fenix Nightly(10/15): 
- I can't tap to select the text boxes after closing the keyboard.
- can't trigger the keyboard when I double-tap the text box selected (TalkBack doesn't even announce the "double-tap to activate" command after reading the text box.
[Recording](https://drive.google.com/file/d/1oPIff3yV_SHR3nGVt3uuXPOsiN-weZN5/view?usp=sharing)

I've compared it to Fennec 68.2b1 and to me it seems the same. But, if this is a different issue, I'll happily open a new ticket.
